### PR TITLE
Add rise hexagons

### PR DESCRIPTION
**Pull request by kalendar**

### CODE REVIEW

File: meta/rise.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "rise",
+  "author": "david wiley",
+  "license": "CC0",
+  "raster": "http://hexb.in/hexagons/dat.png",
```
Comment (pkq) [inline meta/rise.json]: The file name here should match the name of the .png you are submitting.

File: meta/rise.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  "vector": "http://hexb.in/vector/dat.svg",
```
Comment (pkq) [inline meta/rise.json]: Same as above. This should be rise.svg

File: meta/rise.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+  "description": "this is optional!",
```
Comment (pkq) [inline meta/rise.json]: Please add a description, so others know what rise is.

File: meta/rise.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+  "order_online_url": "this is optional. should be a link to where people can buy the sticker online"
```
Comment (pkq) [inline meta/rise.json]: Please add a valid URL or remove this line (and the comma on the previous line) if none.

**Review by pkq — CHANGES_REQUESTED** [review]
Thanks for your submission. Please make the requested changes to rise.json.

**Comment by kalendar** [comment]
The changes have been made - thanks!

**Review by pkq — APPROVED** [review]
Thanks for the prompt response!

**Pull request merged**